plugins.txt

```diff
@@ -1,28 +1,28 @@
 %s An update is available=%s: An update is available.
-anonymous=
-Author=
-Check For Updates=
-Click here to install version %s=
-DOWNGRADE TO VERSION=
-Enter URL of remote plugin file or local plugin file=
-ERROR=
-Install Plugin=
-install=
-Installed Plugins=
-Installing Update=
-need check=
-Next=
-No release notes available=
-Plugin File=
-Plugin Update Check=
-Plugin=
-Release Notes=
-remove=
-Select local plugin file=
-Stable=
-STALE=
-Support Thread=
-Uninstall=
-Update All Plugins=
-Upgrade=
-View Release Notes=
\ No newline at end of file
+anonymous=anonym 
+Author=Autor
+Check For Updates=Suche nach Aktualisierungen
+Click here to install version %s=Klicke hier um Version %s zu installieren
+DOWNGRADE TO VERSION=DOWNGRADE AUF VERSION
+Enter URL of remote plugin file or local plugin file=Gebe URL des Plugins ein oder wähle Datei eines lokalen Plugins aus
+ERROR=FEHLER
+Install Plugin=Installiere Plugin
+install=Installieren
+Installed Plugins=Installierte Plugin
+Installing Update=Installiere Aktualisierung
+need check=benötigt Überprüfung
+Next=Nächster
+No release notes available=Keine Veröffentlichungshinweise vorhanden
+Plugin File=Plugin Datei
+Plugin Update Check=Plugin 
+Plugin=Plugin
+Release Notes=Versionshinweise
+remove=entfernen
+Select local plugin file=Wähle lokale Plugin Datei
+Stable=stabil
+STALE=VERALTET
+Support Thread=Unterstützungs Thema
+Uninstall=Deinstallieren
+Update All Plugins=Aktualisiere alle Plugins
+Upgrade=auf eine höhere Stufe bringen
```

Comment:
oder einfach nur Aktualisieren? 
Ich weiß nicht ob in diesem Kontext der Unterschied zwischen update/upgrade relevant ist, aber wenn ja fällt mir auch grad nix passenderes ein...

Reply:
Aktualisieren wäre gut.

Reply:
Ich kann leider nicht im GUI sehen, ob da ein echter Unterschied zwischen Update und Upgrade ist. Falls nicht, wäre ich auch für "Aktualisieren".

Reply:
Muss nicht alles perfekt am anfang sein, plausibilität prüfe ich dann nächste woche.